environments/echis/public.yml

```diff
@@ -61,7 +61,7 @@ localsettings:
   EMAIL_SMTP_HOST: 'smtp.gmail.com'
   EMAIL_SMTP_PORT: 587
   EMAIL_USE_TLS: yes
-  ENABLE_SOFT_ASSERT_EMAILS: no
+  ENABLE_SOFT_ASSERT_EMAILS: False
```

Comment:
These two are the same. (`no` is a boolean literal equivalent to `False` in YAML: see https://docs.ansible.com/ansible/2.4/YAMLSyntax.html)

Reply:
Ok, in that case I'll close out this PR